src/Text/Pandoc/CSS.hs

```diff
@@ -10,7 +10,7 @@ import Text.Parsec.String
 ruleParser :: Parser (String, String)
 ruleParser = do
     p <- many1 (noneOf ":")  <* char ':'
-    v <- many1 (noneOf ":;") <* char ';' <* spaces
+    v <- many1 (noneOf ":;") <* (optional $ char ';') <* spaces
```

Comment:
Right... so it should be:

    ruleParser :: Parser (String, String)
    ruleParser = do
        p <- many1 (noneOf ":")  <* char ':'
        v <- many1 (noneOf ":;") <* spaces
        return (trim p, trim v)

    styleAttrParser :: Parser [(String, String)]
    styleAttrParser = liftM2 (:)
      (ruleParser <* char ';' <* spaces) $ many
      $ ruleParser <* (optional $ char ';') <* spaces

Or is there a nicer way to write that?